### PR TITLE
Add how-to-contribute section and code of conduct to README

### PR DESCRIPTION
**Pull request by bion**

### CODE REVIEW

**Review by ndmckinley — APPROVED** [review]

**Review by neel-a-mehta — APPROVED** [review]

**Comment by neel-a-mehta** [comment]
FYI this fixes #92

**Review by hmhsu — APPROVED** [review]

**Review by kendrickt — APPROVED** [review]

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
```
Comment (carolinedanzi) [inline code_of_conduct.md:42]: question: Who is considered a "Community Leader"?

**Review by carolinedanzi — APPROVED** [review]
Thank you for putting this together, it's really nice!

File: code_of_conduct.md (hunk 1 above)
Comment (bion) [reply code_of_conduct.md:42]: All the fellows are community leaders for the purpose of this document and the duration of the fellowship.

**Review by natsid — APPROVED** [review]
Thank you for adding this

**Review by AWBaumann — APPROVED** [review]

File: code_of_conduct.md (hunk 1 above)
Comment (gcapiel) [reply code_of_conduct.md:42]: Thanks for doing this.  LGTM

**Pull request merged**